apps/typegpu-docs/tsconfig.json

```
@@ -5,6 +5,10 @@
     "jsxImportSource": "react",
     "types": ["@webgpu/types"]
   },
-  "include": [".astro/types.d.ts", "**/*"],
+  "include": [
+    ".astro/types.d.ts",
+    "**/*",
+    "../../packages/typegpu-geometry/src/lines/indices.ts"
+  ],
```

This sometimes happens by mistake, gets automatically added by the IDE.

```suggestion
  "include": [".astro/types.d.ts", "**/*"],
```